modules/govuk/manifests/nodes/s_postgresql_db_admin_base.pp

```diff
@@ -0,0 +1,79 @@
+# == Class: govuk::nodes::s_postgresql_db_admin_base
+#
+# This machine class defines the common configuration used to
+# administer the app-specific DB-admin machines for PostgreSQL
```

Comment:
```suggestion
# This class installs the tooling for a node to be a PostgreSQL
# db_admin machine
```

Common configuration doesn't seem quite way to phrase it, since there's more than configuration